umbraco/Umbraco-CMS
# Migrated RenderNoContentController + related

### Details
- Migrated the RenderNoContentController
- Migrated routing of RenderNoContentController

### Notes
The functionality that rewrites the request to use this endpoint, if no content is found, is not part of this PR!


### Tests
- https://localhost:44331/umbraco/UmbNoContent
    - Check the page is rendered correct by accessing the direct URL If no content
    - Check the page is redirecting to "/" if content
